src/libs/localStore/userPersistedStorage.ts

```diff
@@ -5,7 +5,7 @@ import { normalizeFqdn } from '/libs/functions/stringHelpers'
 import { storeData, getData } from './storage'
 
 export enum UserPersistedStorageKeys {
-  BackupConfig = 'AmiralAppBackupConfig'
+  LocalBackupConfig = 'AmiralAppLocalBackupConfig'
```

Comment:
Why did you chance this? Won't we will lose some data on users' devices if we rename this without any migration methods? Or maybe this was not used yet?

Reply:
This was not used yet.